### PR TITLE
Add Outline to EasyPanel!

### PR DESCRIPTION
**Pull request by DrMxrcy**

### CODE REVIEW

File: templates/outline/index.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      projectName: input.projectName,
+      serviceName: input.redisServiceName,
+      password: redisPassword,
+    },
+  });
+  services.push({
+    type: "app",
+    data: {
+      projectName: input.projectName,
+      serviceName: "outline-minio",
```
Comment (deiucanta) [inline templates/outline/index.ts:78]: Can you make this configurable?

File: templates/outline/index.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  services.push({
+    type: "app",
+    data: {
+      projectName: input.projectName,
+      serviceName: input.appServiceName,
+      env: [
+        `PGSSLMODE=disable`,
+        `SECRET_KEY=${appKey}`,
+        `UTILS_SECRET=${appUtilsSecret}`,
+        `DATABASE_URL=postgres://postgres:${databasePassword}@${input.projectName}_${input.databaseServiceName}:5432/${input.projectName}?sslmode=disable`,
```
Comment (deiucanta) [inline templates/outline/index.ts:26]: remove empty line

File: templates/outline/meta.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        type: "string",
+        title: "Database Service Name",
+        default: "outline-db",
+      },
+      redisServiceName: {
+        type: "string",
+        title: "Redis Service Name",
+        default: "outline-redis",
+      },
+      googleClient: { type: "string", title: "Google Client ID" },
```
Comment (deiucanta) [inline templates/outline/meta.ts:55]: Can you format this similarly to the previous props? multi-line

File: templates/outline/meta.ts [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+      googleClientSecret: { type: "string", title: "Google Client Secret" },
+    },
+  },
+  logo: "logo.png",
+  screenshots: [],
```
Comment (deiucanta) [inline templates/outline/meta.ts:60]: Can you add some screenshots?

**Comment by DrMxrcy** [comment]
All issues Fixed

**Pull request merged**